src/test/java/org/jboss/ejb/client/test/TransactionTestCase.java

```diff
@@ -107,24 +92,25 @@ public static void beforeClass() throws Exception {
     @Before
     public void beforeTest() throws Exception {
         // start a server
-        server1 = new DummyServer("localhost", 6999, SERVER1_NAME, true);
-        server2 = new DummyServer("localhost", 7999, SERVER2_NAME, true);
-        server3 = new DummyServer("localhost", 8999, SERVER3_NAME, true);
-        server4 = new DummyServer("localhost", 9999, SERVER4_NAME, true);
-        server1.start();
-        server2.start();
-        server3.start();
-        server4.start();
-        serverStarted = true;
-        logger.info("Started servers ...");
-
-        server1.register(APP_NAME, MODULE_NAME, DISTINCT_NAME, EchoBean.class.getSimpleName(), new EchoBean(SERVER1_NAME));
-        server3.register(APP_NAME, MODULE_NAME, DISTINCT_NAME, EchoBean.class.getSimpleName(), new EchoBean(SERVER3_NAME));
-        server4.register(APP_NAME, MODULE_NAME, DISTINCT_NAME, EchoBean.class.getSimpleName(), new EchoBean(SERVER4_NAME));
-
-        server2.register(OTHER_APP, MODULE_NAME, DISTINCT_NAME, EchoBean.class.getSimpleName(), new EchoBean(SERVER2_NAME));
-        server3.register(OTHER_APP, MODULE_NAME, DISTINCT_NAME, EchoBean.class.getSimpleName(), new EchoBean(SERVER3_NAME));
-        server4.register(OTHER_APP, MODULE_NAME, DISTINCT_NAME, EchoBean.class.getSimpleName(), new EchoBean(SERVER4_NAME));
+        for (int i = 0; i < 4; i++) {
+            startServer(i, 6999 + (i*100), true);
+        }
+
+        deployStateless(0);
+        deployStateful(0);
+        // don't deploy on server 2
```

Comment:
Maybe an extra information on server 2 being actually server index 1 would make this comment less confusing

Reply:
Done.